knowledge_storm/interface.py

```diff
@@ -458,3 +561,27 @@ def reset(self):
         self.time = {}
         self.lm_cost = {}
         self.rm_cost = {}
+
+
+class Agent(ABC):
```

Comment:
Agent is a buzzy term nowadays. Add a comment to clarify what it means in our project.